dali/test/python/test_operator_readers_coco.py

```diff
@@ -142,7 +142,9 @@ def test_operator_coco_reader_same_images():
             np.testing.assert_array_equal(boxes1.at(0), boxes2.at(0))
             np.testing.assert_array_equal(boxes1.at(0), boxes3.at(0))
 
-@raises(RuntimeError)
+@raises(
+    RuntimeError,
+    glob="Assert on \"HasArgument(name)\" failed: Argument \"preprocessed_annotations_dir\" is not supported by operator \"readers__COCO\".")
```

Comment:
1. Don't include assertion condition in the message.
2. I remember that on one meeting there was a conclusion that putting entire error messages verbatim is excessive.
3. Nitpick: use single quotes, so you don't have to escape the double quotes - it keeps the pattern more readable.
4. Nitpick: glob can be specified with a positional argument - keeps the line shorter
```suggestion
    'Argument "preprocessed_annotations_dir" is not supported by operator *readers*COCO')
```

Reply:
Done